src/database/models/article.js

```diff
@@ -0,0 +1,23 @@
+import dotenv from 'dotenv';
+
+dotenv.config();
+
+export default (sequelize, DataTypes) => {
+  const Article = sequelize.define(
+    'Article',
+    {
+      title: DataTypes.STRING,
+      description: DataTypes.STRING,
+      content: DataTypes.TEXT,
+      image: DataTypes.STRING,
+      slug: DataTypes.STRING,
+      published: { type: DataTypes.BOOLEAN, defaultValue: true },
```

Comment:
I think what we need to have is a `Published Date` field and a `status` enum field with values `DRAFT`, `ACTIVE` and `DEACTIVATED`. Default value should be `DRAFT`

Reply:
I thought as well too that the 'published' should be as you pointed out, but I had to go along with the UML the team agreed on.
However, I will implement your suggestion to learn from its implementation.